api/specs/common/schemas/node-meta-v0.0.1-converted.yaml

```diff
@@ -246,11 +246,11 @@ properties:
           - type
         properties:
           displayOrder:
-            description: >-
-              DEPRECATED: new display order is taken from the item position.
-              This property will be removed.
-            deprecated: true
```

Comment:
you removed that depraction field. why?

Reply:
when I regenerate the spec it gets automatically excluded. No idea what is happening here